arctic/scripts/arctic_delete_library.py

```diff
@@ -28,7 +28,7 @@ def main():
     if not opts.library:
         parser.error('Must specify the full path of the library e.g. arctic_jblackburn.lib!')
 
-    print "Deleting: %s on mongo %s" % (opts.library, opts.host)
+    print("Deleting: %s on mongo %s" % (opts.library, opts.host))
```

Comment:
I don't think python 2.7 supports print function without the future import, [as documented here](https://docs.python.org/2/library/functions.html#print). The brackets here are treated as grouping (for 2.7), which has no effect in this case.
Anyway, it works, so that'll do :)